### ISSUE TITLE
extensions suddenly vanished

### ISSUE THREAD

**Issue by michaelaye** [ipython-contrib/jupyter_contrib_nbextensions#1356]
I'm using the nbextensions with pleasure for quite some time, and today's conda update has robbed me of them, so I'm sad. ;)
I'm now having these version on my macOS 10.13.6:

```bash
(py36) └─❱❱❱ conda list nbextensions                                           +8970 19:19 ❰─┘
# packages in environment at /Users/klay6683/miniconda3/envs/py36:
#
# Name                    Version                   Build  Channel
jupyter_contrib_nbextensions 0.5.0                    py36_0    conda-forge
jupyter_nbextensions_configurator 0.4.0                    py36_0    conda-forge
```

and the symptom from which I'm inferring that they are inactive are 2-fold:

1. I usually have the dashboard directory listing FILTER active, that's not there anymore
2. As I have the configurator package installed, giving me a notebook tab where I can configure the extensions, that's gone as well.

Here's the weird stuff:

1. the configurator notes that it is loaded, as per jupyter log:

```bash
(py36) └─( 3 ) ❱❱❱ jn .                                                        +8973 19:17 ❰─┘
[I 19:17:16.918 NotebookApp] [nb_conda_kernels] enabled, 3 kernels found
[I 19:17:18.607 NotebookApp] Loading IPython parallel extension
[I 19:17:18.690 NotebookApp] [jupyter_nbextensions_configurator] enabled 0.4.0
[I 19:17:18.796 NotebookApp] JupyterLab extension loaded from /Users/klay6683/miniconda3/envs/py36/lib/python3.6/site-packages/jupyterlab
[I 19:17:18.797 NotebookApp] JupyterLab application directory is /Users/klay6683/miniconda3/envs/py36/share/jupyter/lab
[I 19:17:19.331 NotebookApp] Serving notebooks from local directory: /Users/klay6683/Dropbox/src
[I 19:17:19.331 NotebookApp] The Jupyter Notebook is running at:
[I 19:17:19.331 NotebookApp] https://(MacL2938.local or 127.0.0.1):8901/
[I 19:17:19.331 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 19:17:41.655 NotebookApp] 302 GET / (::1) 2.77ms
```
and I'm not seeing any other error message? I'm puzzled... any hints how I can debug this?

**Comment by michaelaye** [comment]
I have confirmed that this issue goes away when downgrading notebook to 3.6 from 3.7. Should I close here and submit an issue at the `jupyter/notebook` repo?

**Comment by astroboylrx** [comment]
Encounter this issue, too. Spent hours on it without any progress. Downgrading solved it.

**Comment by juhasch** [comment]
Seems notebook 5.7 does not include `jqueryui` anymore. PR #1359 should fix this for the tree-filter extension.

**Comment by hepingtao** [comment]
Exactly the same issue to me, a waste of almostly one day, a terrible experience to me so much. And to be alright by following the solution from all guys above finnally, many thanks.

**Comment by iipr** [comment]
Same thing happened to me. I spent a few hours trying to figure out what was going on, and eventually this issue and doing the downgrade worked for me. I was getting desperate.
Thank you ☺️

**Comment by michaelaye** [comment]
yeah, I have a conda config setting on all my installs now that keeps notebook at 5.6 when I do a `conda update --all`. This is how you can add that to your conda config:

`conda config --env --add pinned_packages notebook=5.6`

The `--env` adds this setting **ONLY** to your currently active environment. That's good because you most likely don't want this to happen everywhere. ;) Just don't forget to regularly check if this issue has been closed and remove the pinning later like so:

`conda config --env --remove pinned_packages notebook=5.6`
